test/blackbox-tests/test-cases/pkg/ocamlformat/building-ocamlformat-dev-tool-fails.t

```diff
@@ -0,0 +1,86 @@
+With a faulty version of OCamlFormat, $ dune fmt is supposed to stop with the
```

Comment:
```suggestion
With a faulty version of OCamlFormat, `dune fmt` is supposed to stop with the
```
Surround the commands in quotes (any type of quotes is fine) so that it's clear where the command ends and the regular text resumes.